src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
 export interface Figure {
-
+  shape: 'triange' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  color: string;
+
+  shape: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
no need for those empty lines in between
```suggestion
  color: string;
  shape: string;
  a: number;
  b: number;
  c: number;
```